### ISSUE TITLE
Support generation of meson build files

### ISSUE THREAD

**Issue by awvwgk** [aradi/fypro#2]
Even if there are known issues for fypp usage in meson (https://github.com/mesonbuild/meson/issues/1348), not all projects require fypp to build. I would be interested to use it to manage both my meson and CMake build files automatically.

Similar to CMake, in meson some problems require boilerplate code to work around (see https://github.com/mesonbuild/meson/issues/5374#issuecomment-830662831 for module installation).

**Comment by aradi** [comment]
Yep, that was the plan, that is should offer the possibility to use templates for different build systems. So your expertise on that would be more than welcome. Of course, without fypp, the fytest framework would not work for the user...
